### PR TITLE
GCI-203: Comparison of String object using != in Module.extensionsMatchNames()

### PR DESCRIPTION
**Pull request by Harshan01**
## GCI-203: Corrected Incorrect Comparison of String object using != in Module.extensionsMatchNames() using String.equals()

see https://issues.openmrs.org/browse/GCI-203

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

  No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [**code style**](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.

  No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

- [ ] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`

- [ ] I ran `mvn clean package` right before creating this pull request and
  added all formatting changes to my commit.

  No? -> execute above command

- [ ] All new and existing **tests passed**.

  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

- [x] My pull request is **based on the latest changes** of the master branch.

  No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

**Comment by merovingienne** [comment]
This looks good. @dkayiwa could we merge this? :)

File: api/src/main/java/org/openmrs/module/Module.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
 import java.util.Properties;
 import java.util.Set;
 import java.util.Vector;
+import java.lang.String;
```
Comment (gayanW) [inline api/src/main/java/org/openmrs/module/Module.java]: `java.lang.*` is imported by default. So there is no need to explicitly import `java.lang.String`

**Comment by dkayiwa** [comment]
Can you squash these commits into one?

**Comment by Harshan01** [comment]
I tried to do it. But when I push it to remote, it shows "unable to push refs" error. Any suggestions?

**Comment by gayanW** [comment]
@Harshan01 
I think you have to force push. But be careful when doing so.

https://stackoverflow.com/questions/17670717/want-to-squash-multiple-commits-in-github

**Pull request closed without merging**